File: Praktikum3/PRAK304-2010817210023-MuhammadAuliaAkbar.py
angka = int(input("Input: "))

print("Output:", end=" ")

if angka == 0:
    print("Nol")
elif angka >= 1 and angka <= 9:
    print("Satuan")
elif angka >= 10 and angka <= 19:
    print("Belasan")
elif angka >= 20 and angka <= 99:
    print("Puluhan")
else:
    print("Anda Menginput Melebihi Limit Bilangan")

File: Praktikum3/PRAK304-2010817210023-MuhammadAuliaAkbar.c
#include <stdio.h>

int main() {
    int angka;

    printf("Input: ");
    scanf("%d", &angka);

    printf("Output: ");
    if (angka == 0) {
        printf("Nol\n");
    } else if (angka >= 1 && angka <= 9) {
        printf("Satuan\n");
    } else if (angka >= 10 && angka <= 19) {
        printf("Belasan\n");
    } else if (angka >= 20 && angka <= 99) {
        printf("Puluhan\n");
    } else {
        printf("Anda Menginput Melebihi Limit Bilangan\n");
    }

    return 0;
}